src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -107,17 +108,15 @@ internal sealed partial class SyntaxTreeIndex
 
                             containsThisConstructorInitializer = containsThisConstructorInitializer || syntaxFacts.IsThisConstructorInitializer(token);
                             containsBaseConstructorInitializer = containsBaseConstructorInitializer || syntaxFacts.IsBaseConstructorInitializer(token);
+                            containsGlobalKeyword = containsGlobalKeyword || syntaxFacts.IsGlobalNamespaceKeyword(token);
 
-                            if (syntaxFacts.IsIdentifier(token) ||
-                                syntaxFacts.IsGlobalNamespaceKeyword(token))
```

Comment:
same ugly hack where when we encountered this specific keyword, we added it to the identifier set for the file.  this was because we didn't use to store any other token information as a bit in the index.  but we changed that up ages ago, and it's trivial to store this info.